hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/throttle/DispatchUsageManager.java

```diff
@@ -79,6 +79,23 @@ public void screenForCapacity(@NonNull final Dispatch dispatch) throws ThrottleE
         }
     }
 
+    /**
+     * Tracks usage of the given dispatch after processing by a handler. The only current example for now is the
+     * ops duration throttle.  This method presumes that the state for the throttles has already been loaded by a call
+     * to {@link DispatchUsageManager#screenForCapacity}
```

Comment:
Would strongly prefer to check this condition in the code. `dispatch.throttleStrategy() == ON` doesn't seem sufficient. Something like `if (!alreadyLoaded()) throw ThrottleException`